config/locales/de.yml

```diff
@@ -264,6 +269,7 @@ de:
   change_my_password: Ihr Passwort ändern
   change_password: Ihr Passwort ändern
   changed_by: Geändert durch
+  checklist: Abhak-Liste
```

Comment:
"Abhak Liste" sounds wierd for me, allthough I know she called it like this in the Mail, but what do you think about Checkliste ?